libphal/phal_dump.C

```diff
@@ -185,48 +222,22 @@ struct pdbg_target* preCollection(const uint32_t failingUnit)
 	fapi2::ReturnCode fapiRc;
 	P10_EXTRACT_SBE_RC::RETURN_ACTION recovAction;
 
+	struct WriteSbeData data;
+	std::filesystem::path basePath = "/tmp/sbe_data_file";
```

Comment:
file name should be unique , otherwise this is going to be issue with mulitple dump requests. you can follw the same directory structre sbe dump uses.

Reply:
Done.